spec/connection_spec.rb

```diff
@@ -54,8 +54,8 @@ def self.data
     end
 
     it :websocket do
-      expect(WebSocket::Client::Simple).to receive(:connect).with('ws://SERVER_A:123/')
```

Comment:
please also add a test for the default path.

Reply:
It seems that the spec located in L51-L54 tests for all of the default options.

https://github.com/marcelocf/gremlin_client/blob/0d1cb6500926654d64cd0f2c2a9aa0bcb0657558/spec/connection_spec.rb#L51-L54